ui/src/components/filter/components/Label.vue

```diff
@@ -9,15 +9,39 @@
 <script setup lang="ts">
     import {computed} from "vue";
 
-    const props = defineProps({option: {type: Object, required: true}});
+    interface Option {
```

Comment:
I think this is not ideal, as the list is still changing as I go through the code and add more filter options, so possibly make it dynamic in a way? Maybe we can revert changes to all `*.vue` components in `/filters` folder for now, until I get it closer to wrapped up state?

Reply:
Sounds good, will revert.